scripts/e2e_tests/trigger_e2e_tests_pipeline.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [ $# -ne 5 ]; then
+    echo "Usage: $0 <APP_DOWNLOAD_URL> <TRIGGERING_WORKFLOW_ID> <TRIGGERING_WORKFLOW_WAITING_JOB_NAME> <TRIGGER_ANDROID_WORKFLOW> <TRIGGER_IOS_TRIGGER_ANDROID_WORKFLOW>"
+    exit 1
+fi
+
+ANDROID_APP_DOWNLOAD_URL=$1
+TRIGGERING_WORKFLOW_ID=$2
+TRIGGERING_WORKFLOW_WAITING_JOB_NAME=$3
+TRIGGER_ANDROID_WORKFLOW=$4
+TRIGGER_IOS_WORKFLOW=$5
+
+TRIGGERED_PIPELINE=$(curl --request POST \
+  --url https://circleci.com/api/v2/project/gh/twilio/twilio-verify-passkeys-tests/pipeline \
+  --header "Circle-Token: $CIRCLE_TOKEN" \
+  --header "Content-Type: application/json" \
+  --data '{
+    "branch":"sfierro/approve-job-in-workflow",
```

Comment:
I'll update the branch once it gets merged https://github.com/twilio/twilio-verify-passkeys-tests/pull/16